src/app/wallet/wallet/send/send.service.ts

```diff
@@ -121,29 +144,13 @@ export class SendService {
     *
     */
   getSendRPCCall(input: string, output: string) {
-    // real send (external)
-    if (input === 'balance' && output === 'balance') {
+
+    input = input.replace('_balance', '').replace('balance', 'part');
```

Comment:
Maybe we should do this replacing earlier on.
balance -> part, that way the TransactionBuilder is more consistent with what the RPC API expects.